data/postData.js

```diff
@@ -0,0 +1,41 @@
+//write functions here to take data and push
+
+const posts = [
+    {
+        timestamp: new Date("2021-05-18T16:00:00Z"),
+        type: "picture",
```

Comment:
use the same enums here and export the enums from here as they are properties of data and not component

Reply:
true